_docs_integrate/attribute-introduction.md

```diff
@@ -138,15 +138,27 @@ The term ThirdPartyRelationshipAttribute is always descriptive for a third party
 
 ## Attribute management options
 
-The IdentityAttributes and RelationshipAttributes were previously introduced. Regardless of whether an Attribute is an IdentityAttribute or a RelationshipAttribute, various operations can be performed with Attributes. These are described on separate documentation pages. In the following, the most important Attribute management options are briefly described.
+The IdentityAttributes and RelationshipAttributes were previously introduced. Regardless of whether an Attribute is an IdentityAttribute or a RelationshipAttribute, various operations can be performed with Attributes. These are described on separate documentation pages. In the following, the Attribute management options are briefly described.
 
 ### Create Attributes for yourself
 
 Obviously, it is not possible to work with Attributes that have not yet been created. Therefore, the most important feature is the [creation of Attributes for yourself]({% link _docs_integrate/create-attributes-for-yourself.md %}).
 
-### Exchange Attributes with peers
+### Share Attributes with peer
 
-Attributes can not only be managed locally, but can also be exchanged with peers. There is the option of [sharing Attributes with peers]({% link _docs_integrate/share-attributes-with-peer.md %}), [reading Attributes from peers]({% link _docs_integrate/read-attributes-from-peer.md %}), [creating Attributes for peers]({% link _docs_integrate/create-attributes-for-peer.md %}) and [proposing Attributes to peers]({% link _docs_integrate/propose-attributes-to-peer.md %}).
+It is possible to [share Attributes with peers]({% link _docs_integrate/share-attributes-with-peer.md %}). The Identity can share own Attributes and also RelationshipAttributes that exist in the context of a Relationship between it and a third party.
```

Comment:
In the second sentence I find the term "third party" tricky, as it always depends on the perspective who the third party is. Instead, I'd rather shift the focus to the fact that using a ShareAttributeRequestItem you can only share already existing Attributes. These could either be RepositoryAttributes or RelationshipAttributes from a Relationship with another peer. How about:

"An Identity can either share IdentityAttributes about itself or RelationshipAttributes from a Relationship with another peer, provided the `confidentiality` allows for it."

If you want to, you can also add links to IdentityAttribute and RelationshipAttribute in the Data Model Overview, since we mention a property.